docs/src/examples/Gol, Lazar & Belta (2013).jl

```diff
@@ -160,8 +161,6 @@ x2 = [xu.x[j][2] for j in eachindex(xu.x)]
 scatter!(p, x1, x2)
 
 # ### References
-# <a id="1">[1]</a> 
-# Gol, E. A., Lazar, M., & Belta, C. (2013). Language-guided controller synthesis for linear systems. IEEE Transactions on Automatic Control, 59(5), 1163-1176.
-#
-# <a id="2">[2]</a>
-# Bemporad, A., & Morari, M. (1999). Control of systems integrating logic, dynamics, and constraints. Automatica, 35(3), 407-427.
\ No newline at end of file
+# 
+# 1. Gol, E. A., Lazar, M., & Belta, C. (2013). Language-guided controller synthesis for linear systems. IEEE Transactions on Automatic Control, 59(5), 1163-1176.
+# 1. Bemporad, A., & Morari, M. (1999). Control of systems integrating logic, dynamics, and constraints. Automatica, 35(3), 407-427.
```

Comment:
Shouldn't this 1. be 2.?

Reply:
It updates to the correct numbers when the .md is generated
![image](https://user-images.githubusercontent.com/12188889/107206975-07e62680-6a00-11eb-91e7-9b6ea0b9eb5b.png)